app/src/main/java/com/mrudultora/colorpickerapp/ColorPickerDialogsFragment.java

```diff
@@ -148,7 +150,8 @@ public void onDirectColorSelected(int color, int position) {
                         defaultColor = color;
                         tabLayout.setBackgroundColor(color);
                     }
-                });
+                })
+                .build();
```

Comment:
Can we use directly `show()` instead of `build()` because `show()` is calling `build().show()`.

Reply:
Yes, I did make build as a separate method so if the caller need to do some modifications before showing the dialog.

Should i remove it?

Reply:
No no, it's good. I will take a final look and then merge it.